src/bcif.jl

```diff
@@ -0,0 +1,622 @@
+export decode_column, columns_to_dict, datablock_to_dict
+using LinearAlgebra
+import MsgPack
+
+
+"""
+    read(input::IO, ::Type{BCIFFormat}, structure_name::AbstractString="", remove_disorder::Bool=false, read_std_atoms::Bool=true, read_het_atoms::Bool=true, run_dssp::Bool=false, run_stride::Bool=false)
+
+A function to read a binary CIF file from MolStar and extract the list of attributes and their compressed bytes.
+"""
```

Comment:
I think this needs to be on the line above the function or it won't "attach" for REPL help/docs.